metacom.d.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Metacom
+ */
+export class Metacom {
+  /** url */
+  url: string
+  /** socket */
+  socket: WebSocket
+  /** api */
+  api: any
+  /** callId */
+  callId: number
+  /** calls */
+  calls: Map<any, any>
+  /**
+   * constructor
+   * @param url host
+   */
+  constructor(url: string)
+  /**
+   * ready
+   */
+  ready(): Promise
+  /**
+   * load folder
+   * @param group
+   */
+  load(...group: string): Promise<void>

```

Comment:
```suggestion
  load(...group: string[]): Promise<void>

```

Reply:
You can add fixup to this PR, to do this just add this changes locally and run following:
```
git add -A
git commit --fixup=bb2dfe4db75b533617c9b9f3468dc89b22fa6e2f
```